apps/voting/app/src/screens/VoteDetail.js

```diff
@@ -0,0 +1,276 @@
+  IconCheck,
+  IconTime,
+  Split,
+  Timer,
+  textStyle,
+  useLayout,
+  useTheme,
+} from '@aragon/ui'
+import { useAppState } from '@aragon/api-react'
+import AppBadge from '../components/AppBadge'
+import LocalIdentityBadge from '../components/LocalIdentityBadge/LocalIdentityBadge'
+import SummaryBar from '../components/SummaryBar'
+import SummaryRows from '../components/SummaryRows'
+import VoteActions from '../components/VoteActions'
+import VoteStatus from '../components/VoteStatus'
+import VoteText from '../components/VoteText'
+import { percentageList, round, safeDiv } from '../math-utils'
+import { formatDate } from '../utils'
+import { getQuorumProgress } from '../vote-utils'
+import { VOTE_NAY, VOTE_YEA } from '../vote-types'
+
+const DEFAULT_DESCRIPTION =
+  'No additional description has been provided for this proposal.'
+
+function VoteDetail({ vote, onBack, onVote, onExecute }) {
+  const theme = useTheme()
+  const { layoutName } = useLayout()
+  const { tokenSymbol } = useAppState()
+
+  const { data, numData, voteId, connectedAccountVote } = vote
+  const { minAcceptQuorum, supportRequired, yea, nay } = numData
+  const { creator, endDate, open, metadata, description } = data
+  const quorumProgress = getQuorumProgress(vote)
+  const totalVotes = yea + nay
+  const votesYeaVotersSize = safeDiv(yea, totalVotes)
+  const votesNayVotersSize = safeDiv(nay, totalVotes)
+  const [yeaPct, nayPct] = percentageList(
+    [votesYeaVotersSize, votesNayVotersSize],
+    2
+  )
+  const youVoted =
+    connectedAccountVote === VOTE_YEA || connectedAccountVote === VOTE_NAY
+
+  const handleVoteNo = useCallback(() => {
+    onVote(voteId, VOTE_NAY)
+  }, [onVote, voteId])
+  const handleVoteYes = useCallback(() => {
+    onVote(voteId, VOTE_YEA)
+  }, [onVote, voteId])
+
+  const handleExecute = useCallback(() => {
+    onExecute(voteId)
+  }, [onExecute, voteId])
+
+  return (
+    <React.Fragment>
+      <Bar>
+        <BackButton onClick={onBack} />
+      </Bar>
+      <Split
+        primary={
+          <Box>
+            <div
+              css={`
+                display: flex;
+                justify-content: space-between;
+              `}
+            >
+              <AppBadge>App Badge</AppBadge>
+              {youVoted && (
+                <div
+                  css={`
+                    display: inline-grid;
+                    grid-template-columns: auto auto;
+                    grid-gap: ${0.5 * GU}px;
+                    align-items: center;
+                    justify-content: center;
+                    height: 20px;
+                    width: auto;
+                    border-radius: 100px;
+                    padding: 0 ${1 * GU}px;
+                    background: ${theme.infoSurface.alpha(0.08)};
+                    color: ${theme.info};
+                    ${textStyle('label2')};
+                  `}
+                >
+                  <IconCheck size="tiny" /> Voted
+                </div>
```

Comment:
@dizzypaty Should this be a `<Tag size="small">`?

Currently using that displays:

<img width="851" alt="Screen Shot 2019-08-12 at 1 11 36 AM" src="https://user-images.githubusercontent.com/4166642/62840635-331e7700-bc9e-11e9-9cec-9d6e1ede154b.png">

Which is quite different than the [design](https://www.figma.com/file/O09HJRCFDwDTni4VE3tiRvwI/Voting?node-id=375%3A866).

Reply:
It should use the large tag size.

Reply:
Also, I know this is not correct in many of the designs, but the vote title 'Vote #24' should use the text style `Title 2`.